tutorials/integrations/local-subsquid.md

```diff
@@ -0,0 +1,607 @@
+
+      if (i.address === contractAddress && i.kind === "evmLog"){
+          if (i.transaction){
+            const { from, to, value } = events.Transfer.decode(i.evmLog)
+            transfers.push(new Transfer({
+              id: `${String(c.header.height).padStart(10, '0')}-${i.transaction.hash.slice(3,8)}`,
+              block: c.header.height,
+              from: from,
+              to: to,
+              value: value.toBigInt(),
+              timestamp: BigInt(c.header.timestamp),
+              txHash: i.transaction.hash
+            }))
+          }
+      }
+    }
+   }
+   await ctx.store.save(transfers)
+});
+```
+
+??? code "View the complete `processor.ts` file"
+
+    ```js
+    import { TypeormDatabase } from "@subsquid/typeorm-store";
+    import { EvmBatchProcessor } from "@subsquid/evm-processor";
+    import { Transfer } from "./model";
+    import { events } from "./abi/MyTok";
+
+    const contractAddress =
+      "0xc01Ee7f10EA4aF4673cFff62710E1D7792aBa8f3".toLowerCase();
+    const processor = new EvmBatchProcessor()
+      .setDataSource({
+        chain: "http://localhost:9944",
+        archive: "http://localhost:8080",
+      })
+      .addLog(contractAddress, {
+        filter: [[events.Transfer.topic]],
+        data: {
+          evmLog: {
+            topics: true,
+            data: true,
+          },
+          transaction: {
+            hash: true,
+          },
+        },
+      });
+
+    processor.run(new TypeormDatabase(), async (ctx) => {
+      const transfers: Transfer[] = [];
+      for (let c of ctx.blocks) {
+        for (let i of c.items) {
+          if (i.address === contractAddress && i.kind === "evmLog") {
+            if (i.transaction) {
+              const { from, to, value } = events.Transfer.decode(i.evmLog);
+              transfers.push(
+                new Transfer({
+                  id: `${String(c.header.height).padStart(
+                    10,
+                    "0"
+                  )}-${i.transaction.hash.slice(3, 8)}`,
+                  block: c.header.height,
+                  from: from,
+                  to: to,
+                  value: value.toBigInt(),
+                  timestamp: BigInt(c.header.timestamp),
+                  txHash: i.transaction.hash,
+                })
+              );
+            }
+          }
+        }
+      }
+      await ctx.store.save(transfers);
+    });
+    ```
+
+Now we've taken all of the steps necessary and are ready to run our indexer!
+
+### Run the Indexer {: #run-indexer }
+
+To run our indexer, we're going to run a series of `sqd` commands:
+
+1. Build our project
+
+    ```
+    sqd build
+    ```
+
+2. Launch the database:
+
+    ```
+    sqd up
+    ```
+
+3. Remove the database migration file that comes with the EVM template and generate a new one for our new database schema:
+
+    ```
+    sqd migration:clean
+    sqd migration:generate
+    ```
+
+4. Launch the processor:
+
+    ```
+    sqd process
```

Comment:
Was not working for  me. Initially I thought it was because I was not using `--sealing XXXX` but then it continued not working